_layouts/homepage.html

```diff
@@ -24,15 +24,18 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
     <div class="u-marginBottom25">
       {%
         include components/universal/Alert/index.html
-        title="Reminder"
-        message="The Medium Term Budget Policy Statement (MTBPS), a Cabinet policy statement tabled in Parliament about three months before the annual budget, will be streamed live from the vulekamali homepage on Wednesday, the 25th of October."
+        title="Please note"
+        message="...pending description"
```

Comment:
Can this have links?

"The full Medium Term Budget Policy Statement is [available on National Treasury's website](URL). The MTBPS documents and data will be added to vulekamali.gov.za soon."

Reply:
Including the link inline adds a bit of complexity. Can I perhaps just put the actually URL as text in the message? Maybe in brackets?

Reply:
Please drop the include completely, and just use raw HTML.